Dockerfile

```diff
@@ -38,6 +38,7 @@ RUN addgroup vault && \
 
 # Set up certificates, base tools, and software.
 RUN set -eux && \
+    apk update && apk upgrade libretls && \
```

Comment:
Should we add a ticket to remind us to check whether we can remove this (i.e. base image is updated)?